### PR TITLE
Ane Marte's Technigo Project "Spis Meg"

### PR DESCRIPTION
**Pull request by amskogesal**
I've started to create a website to provide people recipes based on what ingredients they already have in their fridge (using check boxes). This was a lot more complicated than I imagined, so please consider this a start. 

I struggled a lot with scaling of the image. It got better after some experimenting, but still not completely happy with it.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -4,18 +4,47 @@
...
+      <h2>Kaster du ofte mat? Synes du det er vanskelig å utnytte all maten du har i kjøleskapet?</h2h>
+      
+      <h3>Spis Meg er det digitale oppslagsverket som foreslår oppskrifter basert på hvilke matvarer du allerede har i kjøleskapet</h3>
+
+      <h3>Hvorfor? Hver nordmann kaster gjennomsnittlig 42 kg mat i året. Norge har forpliktet seg til FNs bærekraftmål, og skal innen 2030 halvere matsvinnet per innbygger.
+      Kast mindre mat, spar penger, og gjør noe godt for kloden vår.</h3> 
+
+      
+
+    </div>
```
Comment (HIPPIEKICK) [inline code/index.html:34]: We should indent our code to make it easier to read. Take a look at our [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you wanna learn more :)

```
<img class="main-image" src="image.food.jpg" alt="food">
<div> 
  <h1>SPIS MEG</h1>
  <h2>Kaster du ofte mat? Synes du det er vanskelig å utnytte all maten du har i kjøleskapet?</h2h>
  <h3>Spis Meg er det digitale oppslagsverket som foreslår oppskrifter basert på hvilke matvarer du allerede har i kjøleskapet</h3>
  <h3>Hvorfor? Hver nordmann kaster gjennomsnittlig 42 kg mat i året. Norge har forpliktet seg til FNs bærekraftmål, og skal innen 2030 halvere matsvinnet per innbygger.
      Kast mindre mat, spar penger, og gjør noe godt for kloden vår.</h3> 
</div>

```

File: code/index.html [hunk 2]
```diff
@@ -4,18 +4,47 @@
...
+
+    
+
+    <button type="button">Ingredienser</button>
+
+    <button type="button">Alle oppskrifter</button>
```
Comment (HIPPIEKICK) [inline code/index.html:40]: Nice with some buttons!

File: code/index.html [hunk 3]
```diff
@@ -4,18 +4,47 @@
...
+
+    <label for="browser">Choose your browser from the list:</label>
+<input list="browsers" name="browser" id="browser">
```
Comment (HIPPIEKICK) [inline code/index.html:43]: If you want a drop down menu, try out the `select` element! Read more [here](https://www.w3schools.com/tags/tag_select.asp).

File: code/style.css [hunk 4]
```diff
@@ -1,5 +1,35 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+
 body {
-  background-color: lightblue;
+  background-color: rgba(145, 213, 147, 0.646);
+}
+
+.main-image {
```
Comment (HIPPIEKICK) [inline code/style.css:8]: To make this image not be distorted, you can try adding this to the styling of it:
`object-fit: cover;`
That will crop the image as a passe-partout ⭐

File: code/style.css [hunk 5]
```diff
@@ -1,5 +1,35 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+
 body {
```
Comment (HIPPIEKICK) [inline code/style.css:4]: To make the image go aaaall the way to the edges, you can add `margin: 0;` to get rid of the body's default margin!

File: code/style.css (hunk 4 above)
Comment (HIPPIEKICK) [reply code/style.css:8]: Another idea is to add some opacity to the image, like this:
`opacity: 0.7;`

and then add absolute positioning to your heading - to create a hero image. Note - this is just a suggestion for you to play around with:
```
h1 {
  font-size: 75px;
  font-weight: bolder;
  font-family: Arial, Helvetica, sans-serif;
  color: white;
  position: absolute;
  top: 200px;
  left: 30%;
  text-shadow: 4px 4px goldenrod;
}
```

**Review by HIPPIEKICK — COMMENTED** [review]
Great job with the first website! Forms are not the easiest things to begin with (we have another course for that) but you managed to implement buttons and an input field ⭐ 

A general tip to improve the readability of your text (long lines can be hard to read) is to wrap all your HTML content beneath the image (div, h1, buttons, input etc) in a `<main>` tag and then add some padding to that element:

```
main {
  padding: 20px 100px 100px 100px;
}
```

Try it out and see what you think! Keep up the good work ⭐

**Comment by amskogesal** [comment]
Thank you so much for this very valuable input @HIPPIEKICK 🙏  Much appreciated, I'll definitely give it a try

**Pull request closed without merging**